src/View/mainpage/StructureWidget.py

```diff
@@ -8,7 +8,7 @@
 
 class StructureWidget(QtWidgets.QWidget):
 
-    structure_renamed = QtCore.pyqtSignal(Dataset)  # new PyDicom dataset
+    structure_renamed = QtCore.pyqtSignal(tuple)  # (new_dataset, change_description)
```

Comment:
does it go without saying that the new_dataset, which is of type ``pydicom.dataset.Dataset`` is representing an RT Structure Set?  
At some point, people might want to use Segmentation objects (not something you need to worry about), but my point is that in addition to the python data type involved, in some of the code, it really does matter which DICOM object (SOP Class) is being represented by the pydicom.dataset.Dataset.  In other parts of the code, it really does not matter which SOP Class is involved.